### ISSUE TITLE
parser_error: false positive in "reduce" statement?

### ISSUE THREAD

**Issue by g-back** [abaplint/abaplint#555]
The following statement (that works in my 7.40 system) is shown as an error:

```abap
      opr->material = reduce string( init x type string
                                  for wa in material_data
                                  where (  plnal = opr->plnal and
                                           plnnr = opr->plnnr and
                                           plnty = opr->plnty )
                                  next x = |{ x } { wa-matnr alpha = out }, | ).
```
 Statement does not exist in ABAPv755(or a parser error)(parser_error)

**Comment by larshp** [comment]
yea, looks like a bug, thanks

**Comment by larshp** [comment]
fixed in above commit, I'll release a new version tomorrow

**Comment by larshp** [comment]
2.1.4 released
